requirements.md

```diff
@@ -66,10 +66,65 @@ These are mostly in response to trends we see in issues made
 - Switch between slower HiFi resampler and a fast LoFi one.
 
 ### Needed by Bevy
-Bevy is a widely used game engine build around the ECS model. Because bevy uses ECS it has specific demands from an audio engine.
 
-*TODO*
+Bevy is a popular game engine built around the ECS paradigm.
```

Comment:
(nitpick) lets link to the Bevy ECS guide (probably this one right? https://bevyengine.org/learn/quick-start/getting-started/ecs/) since its required reading to understand this section.